arangod/MMFiles/MMFilesRestReplicationHandler.cpp

```diff
@@ -869,6 +869,8 @@ void MMFilesRestReplicationHandler::handleCommandLoggerFollow() {
                          StringUtils::itoa(dump._lastFoundTick));
   _response->setHeaderNC(TRI_REPLICATION_HEADER_LASTTICK,
                          StringUtils::itoa(state.lastCommittedTick));
+  _response->setHeaderNC(TRI_REPLICATION_HEADER_LASTSCANNED,
```

Comment:
is TRI_REPLICATION_HEADER_LASTTICK == TRI_REPLICATION_HEADER_LASTSCANNED on rocksdb ?

Reply:
Ah now I got it, lastscanned is always set to a nonzero value if we scanned something